gems/pending/util/miq-password.rb

```diff
@@ -174,7 +174,7 @@ def self.generate_symmetric(filename = nil)
     end
   end
 
-  protected
+  # protected
```

Comment:
Overlap with the other PR, I think.  I asked another contributor on that other PR if this protected was needed because protected on class/singleton methods is useless.